relnotes/pkg.migration.md

```diff
@@ -0,0 +1,19 @@
+* Packaging
+
+  * The `OPTS` and `ARGS` variables used when defining packages have become built-ins now, so they should be modified **in-place** (using `OPTS.update()`, `OPTS['xxx']`, and `ARGS.extend()`, `ARGS.append()` or any other functions that mutates the object).
```

Comment:
Is there any reason to prefer `ARGS.extend` versus `ARGS.append`?  What's the expected difference?

Reply:
Append in only for one element, extend will inject a list into the list.

So:

```py
[1, 2].append(3) # [1, 2, 3]
[1, 2].append([3, 4]) # [1, 2, [3, 4]]
[1, 2].extend([3, 4]) # [1, 2, 3, 4]
```